### PR TITLE
Lead DevOps Engineer

### PR DESCRIPTION
**Pull request by madwork**

### CODE REVIEW

File: devops.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Explain how you would create an unified deployment platform to push new release to production with
+existing capistrano architecture (through SSH) or explain how you would build a new deployent
+pipeline.
+
+Requirements:
+
+    * It can be automated (from CI) and triggered manually (when manual actions are required)
+    * It checks CI success before deploy a new release
+    * It keeps track of every deployments
```
Comment (heldersantosmoreira) [inline devops.md:45]: Typo here, `deployment` instead of `deployments` ✂️

Comment (heldersantosmoreira) [reply devops.md:45]: Or `all deployments`.

File: devops.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Lead DevOps Engineer
+
+As DevOps Engineer, you will be responsible for managing our operations, building our delivery
+pipelines and monitoring our platform to ensure the reliability of our services.
+
+Requirements
+
+* Strong background in Linux administration
+* Automation and configuration management
+* Experience with Ruby web frameworks, specifically Ruby on Rails
```
Comment (alecgibson) [inline devops.md:12]: Node.JS experience also a plus

**Comment by alecgibson** [comment]
I agree with Hélder in that it may be weird for people to skip submitting any actual evidence of being able to set up a deployment pipeline. That being said, I do wonder if we should make it more generic? Or maybe it's fine being specific to eg Chef, given that that's what we use?

File: devops.md (hunk 2 above)
Comment (madwork) [reply devops.md:12]: Node what?!

Comment (emmanuelnataf) [reply devops.md:12]: 😂

File: devops.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Requirements:
+
+    * It can be automated (from CI) and triggered manually (when manual actions are required)
+    * It checks CI success before deploy a new release
+    * It keeps track of every deployments
+    * It queues deployment to avoid conflicts
+    * It allows quick rollback
+    * It ensures deployment integrity through process monitoring
+
+#### Question 4 (optional)
```
Comment (joaodiogocosta) [inline devops.md:50]: (as mentioned in the meeting, just for reference)

Should we make this Q mandatory?

File: devops.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+#### Question 2
+
+* Describe your experience with cloud platforms such as Google Cloud, Amazon AWS, DigitalOcean, etc.
+* Describe your experience with provisioning tools such as Chef, Puppet, Ansible, etc.
+* Describe your experience with monitoring tools such as Stackdriver, Datadog, Nagios, etc.
+
+#### Question 3
+
+Explain how you would create an unified deployment platform to push new release to production with
```
Comment (joaodiogocosta) [inline devops.md:37]: (as mentioned in the meeting, just for reference)

Should we leave capistrano out and let the candidate share their own perspective and tools of preference?

**Pull request merged**